go/vt/vtctl/reparent.go

```diff
@@ -171,3 +177,24 @@ func commandEmergencyReparentShard(ctx context.Context, wr *wrangler.Wrangler, s
 	}
 	return wr.EmergencyReparentShard(ctx, keyspace, shard, tabletAlias, *waitSlaveTimeout)
 }
+
+func commandTabletExternallyReparented(ctx context.Context, wr *wrangler.Wrangler, subFlags *flag.FlagSet, args []string) error {
+	keyspaceShard := subFlags.String("keyspace_shard", "", "keyspace/shard of the shard that needs to be reparented")
```

Comment:
Unfortunately, changing the command-line interface of TER like this is a breaking change. We also don't want to add a requirement onto external tools like Orc that they must know the keyspace/shard in advance. We can look up the keyspace/shard from the tablet record anyway before calling into the wrangler.